### PR TITLE
docs: add Windows on Snapdragon benchmarks

### PR DESCRIPTION
**Pull request by ShuaiLYU**
Adds a new 'Measured Performance on Windows on Snapdragon (win_arm64)' section to the QNN integration page with end-to-end YOLO26n latencies on a Lenovo Snapdragon X Elite (X1E78100) laptop, comparing the native PyTorch FP32 CPU baseline against the QNN Hexagon NPU path across all six tasks. The existing phone-based Measured Performance section is left unchanged.

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary
Adds a new Windows on Snapdragon benchmark section to the QNN integration docs, comparing YOLO26n CPU and Hexagon NPU performance on `win_arm64` hardware. 🚀

### 📊 Key Changes
- Added a new **Measured Performance on Windows on Snapdragon (`win_arm64`)** section to `docs/en/integrations/qnn.md`.
- Documented benchmark results for multiple YOLO26n tasks: **Detect, Segment, Semantic, Classify, Pose, and OBB**.
- Compared **native PyTorch FP32 CPU** inference against **ONNX Runtime QNN Hexagon NPU** inference on a Snapdragon X Elite Windows 11 laptop.
- Included detailed per-stage latency breakdowns for **preprocessing, inference, and postprocessing**.
- Added benchmark methodology details, including runtime setup, warmup/run counts, device state, and measurement scope. 📈
- Summarized the observed speedups, highlighting that the NPU is generally **2–4x faster** than the CPU baseline across larger image sizes. ⚡

### 🎯 Purpose & Impact
- Improves documentation coverage for **Windows on Snapdragon** deployments, an increasingly relevant edge and desktop AI platform.
- Gives users practical, hardware-specific expectations for running YOLO26 with **QNN acceleration** versus standard CPU inference.
- Helps developers evaluate whether migrating to the **Hexagon NPU path** is worthwhile for their workloads.
- Strengthens the QNN integration docs with clearer cross-platform performance evidence, especially for **ARM64 Windows** users. 🪟

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ShuaiLYU, thank you for submitting a `ultralytics/ultralytics` 🚀 PR! This is an automated message to help with review readiness—an engineer will assist you soon. Please review the checklist below for a smooth integration 😊

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_  — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

File: docs/en/integrations/qnn.md [hunk 1]
```diff
@@ -63,6 +63,22 @@ End-to-end single-image inference for the official YOLO26n models on a Xiaomi 17
...
 - <sup>1</sup> Semantic QNN uses the in-graph ArgMax class-map output from this release, which replaced erratic 123-1065 ms logits decoding with a stable ~49 ms; the GPU remains slightly faster for semantic at 1024px.
 
+## Measured Performance on Windows on Snapdragon (win_arm64)
+
+End-to-end single-image inference for the official YOLO26n models on a Lenovo laptop powered by the Qualcomm Snapdragon X Elite (X1E78100) — Qualcomm Oryon CPU and Hexagon NPU (HTP v73), 32 GB RAM, Windows 11. This Windows-on-Snapdragon comparison runs the native PyTorch FP32 CPU baseline that most desktop developers start from against the QNN Hexagon NPU path. Each cell shows the **total time** (preprocessing + inference + postprocessing, excluding annotation) with the per-stage split beneath it; CPU numbers are PyTorch FP32 (`torch==2.10.0+cpu`) and NPU numbers are ONNX Runtime QNN (`onnxruntime-qnn==2.2.0`, INT8 weights / 16-bit activations).
```
Comment (UltralyticsAssistant) [inline docs/en/integrations/qnn.md]: ⚠️ **HIGH**: The methodology says each cell's total is `preprocessing + inference + postprocessing`, but the stage splits in the table do not add up to the reported totals in multiple rows (for example Detect CPU: `4.3 + 75.2 + 0.1 = 79.6`, not `91.4`). That makes the benchmark definition internally inconsistent and readers cannot tell whether there is missing overhead being counted or the numbers are incorrect. Please either correct the figures or explicitly document the extra time included in the total.

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

Useful addition overall, but the benchmark table currently has a blocking consistency issue: the per-stage latency breakdowns do not match the stated total latencies. Fixing or explaining that discrepancy would make the new section reliable.

💬 Posted 1 inline comment

**Review by glenn-jocher — APPROVED** [review]
Validated the docs build and QNN page diff. The benchmark section now keeps both tables under Measured Performance and clarifies the Windows end-to-end timing.
